dask/dataframe/core.py

```diff
@@ -1317,8 +1338,10 @@ def repartition(
             For convenience if given an integer this will defer to npartitions
```

Comment:
This docstring should probably be updated to recommend `tuple` (needn't be in this PR, though).